CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 
 # Changelog
 
+## [22.05.4] - 2022/06/16
```

Comment:
can you add the block at the bottom of the changelog so that this 22.05.4 header will be a proper link. Just create it above 22.05.3 link. The link should be:
`https://github.com/phac-nml/irida/compare/22.05.3...22.05.4`